db.scheme

```diff
@@ -5,7 +5,8 @@ CREATE TABLE IF NOT EXISTS `person` (
 
 CREATE TABLE IF NOT EXISTS `restaurant` (
  `id`   INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
- `name` TEXT NOT NULL
+ `name` TEXT NOT NULL,
+ `comment` TEXT NOT NULL
```

Comment:
Shouldn't this have empty default?
What happens on an existing db? (seems the edit button doesn't work because there's no such column and after a few tries crashes)